crawler/config/centres.py

```diff
@@ -52,6 +58,7 @@
         "sftp_root_read": "project-heron_alderly-park",
         "biomek_labware_class": BIOMEK_LABWARE_CLASS_KINGFISHER,
         CENTRES_KEY_SKIP_UNCONSOLIDATED_FILES: True,
+        CENTRES_KEY_INCLUDE_IN_BATCH_PROCESS: True,
```

Comment:
Just thinking, a small refactor might keep this file a little more clean: when doing `.get(<key>,True)` would be equivalent to having a default value in all the centres. Similar for `CENTRES_KEY_SKIP_UNCONSOLIDATED_FILES`. What do you think?

Reply:
I already do impose the default value in the code, but you can't control how a dictionary gets used especially how someone accessing values chooses to define the default value, compared to an object with a getter.  I agree though that if we are going to declare that there is a default value, that we should remove the re-definition of those defaults in this file.  If I were spending endless time on this, I'd probably create a Centre object and make it enforce the defaults from this config via property getters.